scripts/cloudbuild_builder.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
I think we should leave these headers alone unless it's a new file

Reply:
It's technically a new file but git recognized it as a rename. I'll change it to 2020.